features/mask/core/src/test/java/org/apache/shardingsphere/mask/algorithm/KeepFirstNLastMMaskAlgorithmTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.mask.algorithm;
+
+import org.apache.shardingsphere.mask.algorithm.cover.KeepFirstNLastMMaskAlgorithm;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Properties;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+/**
+ * KEEP_FIRST_N_LAST_M test.
+ */
+public class KeepFirstNLastMMaskAlgorithmTest {
+    
+    private KeepFirstNLastMMaskAlgorithm keepFirstNLastMMaskAlgorithm;
+    
+    @Before
+    public void setUp() {
+        keepFirstNLastMMaskAlgorithm = new KeepFirstNLastMMaskAlgorithm();
+        keepFirstNLastMMaskAlgorithm.init(initProperties());
+    }
+    
+    private Properties initProperties() {
+        Properties properties = new Properties();
+        properties.setProperty("start-index", "2");
+        properties.setProperty("stop-index", "5");
+        properties.setProperty("replace-char", "*");
+        return properties;
+    }
+    
+    @Test
+    public void testMask() {
+        String result = keepFirstNLastMMaskAlgorithm.mask("abc123456");
+        assertEquals("ab*23456", result);
+    }
+    
+    @Test
+    public void testMaskIfPlainValueIsLess() {
+        String result = keepFirstNLastMMaskAlgorithm.mask("abc");
+        assertEquals("abc", result);
```

Comment:
Hi @zzzwjZhang, you can read ShardingSphere code conduct first - https://shardingsphere.apache.org/community/en/involved/conduct/code/. We recommend using assertThat over assertEquals.

Reply:
sorry,i will be careful next time.

Reply:
Don't worry, I'm sure you'll do a great job.